lib/logic/player/music_player.dart

```diff
@@ -658,7 +658,7 @@ class MusicPlayer extends AudioPlayer {
 
   Future<void> init() async {
     await restoreLastSong();
-    await handler?._init(this);
```

Comment:
Can you add more context to this comment? Also, please prefer putting comments above the lines that they refer to